samples/contrib/pytorch-samples/prerequisites.md

```diff
@@ -0,0 +1,88 @@
+# Prerequisites
+
+Following prerequisites needs to be performed for kubeflow 1.3 or lower.
+
+### Modify KFServing torchserve image
+
+Edit inferenceservice-config configmap
+
+```kubectl edit cm inferenceservice-config -n kubeflow```
+
+Update the following keys under `predictors->pytorch->v2` block
+
+```
+"image": "public.ecr.aws/y1x1p2u5/torchserve-kfs",
```

Comment:
Is this different from 'pytorch/torchserve-kfs:0.4.0'

Reply:
We will raise separate PR for requirement.txt. Once that is done we will change this to pytorch/torchserve-kfs:0.4.0